chat.cpp

```diff
@@ -238,6 +238,14 @@ void Chat::HandleMessage(std::stringstream message, std::string sender) {
         WriteMessage(NextLevelMessage(l));
     } else if (command == "current") {
         WriteMessage(CurrentLevelMessage(_qq.Current()));
+    } else if (command == "random" && sender == Auth::channel) {
+        _timer.Reset();
+        std::optional<Level> l = _qq.Random();
+        if (l) {
+            WriteMessage(std::string("/marker " + l->levelCode + ", submitted by "
```

Comment:
Isn't this done with a method in Twitch right now? Better not duplicate the code, especially since we know this approach doesn't work.

Reply:
There is a stub, but the actual code we've tried is here and in Next.  I suppose we could leave it out.